install_herolib.vsh

```diff
@@ -43,20 +42,34 @@ fn addtoscript(tofind string, toadd string) ! {
 	os.write_file(rc_file, new_content)!
 }
 
-vroot := @VROOT
 abs_dir_of_script := dir(@FILE)
 
-// Reset symlinks if requested
+// Determine the organization name from the current path
```

Comment:
should not do this, we want to make sure the code is on the right sport, everything in herolib is designed around making sure we follow the location convention

once we start allowing this it will give lots of side effects

Reply:
I agree with you, but to do that, we’ll need to update all the imports and symlinks, as I mentioned in my previous comment